app/src/main/java/ar/scacchipa/twittercloneapp/data/model/UserWrapper.kt

```diff
@@ -0,0 +1,7 @@
+package ar.scacchipa.twittercloneapp.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class UserWrapper(
```

Comment:
please, choose a naming format either has Wrapper or not but to have both styles is confusing

Reply:
The identifier was changed to UserWrapperMapper.